apps/atomy-q/API/tests/Feature/Api/V1/AiStatusVisibilityTest.php

```
@@ -0,0 +1,36 @@
+{
+    public function testItDoesNotLeakEndpointUrisOrAuthTokens(): void
+    {
+        config()->set('atomy.ai.mode', AiStatusSchema::MODE_PROVIDER);
+        config()->set('atomy.ai.provider.name', 'hf-public');
+        config()->set('atomy.ai.provider.default_auth_token', 'hf-super-secret-token');
+        config()->set('atomy.ai.endpoints.document.uri', 'https://hf.example.test/document/private');
+        config()->set('atomy.ai.endpoints.document.auth_token', 'hf-document-secret-token');
+
+        Http::fake([
+            'https://hf.example.test/document/private/health' => Http::response(['ok' => true], 200),
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Http' in method 'testItDoesNotLeakEndpointUrisOrAuthTokens'. (undefined)

(StaticAccess)